Content.Server/Atmos/Piping/Binary/EntitySystems/GasValveSystem.cs

```diff
@@ -52,18 +52,30 @@ private void OnActivate(EntityUid uid, GasValveComponent component, ActivateInWo
             if (args.User.InRangeUnobstructed(args.Target) && Get<ActionBlockerSystem>().CanInteract(args.User))
             {
                 Toggle(uid, component);
-                SoundSystem.Play(Filter.Pvs(component.Owner), component._valveSound.GetSound(), component.Owner, AudioHelpers.WithVariation(0.25f));
+                SoundSystem.Play(Filter.Pvs(component.Owner), component.ValveSound.GetSound(), component.Owner, AudioHelpers.WithVariation(0.25f));
             }
         }
 
         public void Set(EntityUid uid, GasValveComponent component, bool value)
         {
             component.Open = value;
+            var appearance = EntityManager.GetComponentOrNull<AppearanceComponent>(component.Owner);
```

Comment:
Why not just TryComp & set appearance data?
Or is there some reason it should only be called if this also has a `NodeContainerComponent`?